### ISSUE TITLE
Unable to import data

### ISSUE THREAD

**Issue by RichardLitt** [HaroldMills/Vesper#155]
I am trying to import data from my Recordings folder, using `http://127.0.0.1:8000/import-recordings/`. The folder that has the data is `/Users/richard/src/NBNC_Archive/Recordings`, while I am running the server from `/Users/richard/src/NBNC_Archive`. I am getting this error:

```
2019-11-13 20:14:37,486 ERROR    Job raised exception. See traceback below.
Traceback (most recent call last):
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/job_runner.py", line 124, in run_job
    command = _create_command(job_info.command_spec)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/job_runner.py", line 223, in _create_command
    return command_class(command_args)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/import_command.py", line 18, in __init__
    self._importer = _create_importer(importer_spec)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/import_command.py", line 36, in _create_importer
    return cls(arguments)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/recording_importer.py", line 48, in __init__
    self.file_parser = _create_file_parser(spec)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/recording_importer.py", line 352, in _create_file_parser
    station_name_aliases = _get_station_name_aliases(spec, stations)
  File "//anaconda3/envs/vesper-0.4.7/lib/python3.6/site-packages/vesper/command/recording_importer.py", line 378, in _get_station_name_aliases
    preset_name))
vesper.command.command.CommandExecutionError: Could not find Station Name Aliases preset "Station Name Aliases".
```

Any idea how I can resolve this?

**Comment by HaroldMills** [comment]
Thanks for reporting this issue, Richard. Vesper currently (as of version 0.4.7) requires that every archive have a Station Name Aliases preset with the name "Station Name Aliases". You can put an empty "Station Name Aliases.yaml" file in a "Station Name Aliases" subdirectory of your archive's "Presets" directory to satisfy this requirement. Shortly, I will also commit a change to the recording importer that eliminates the requirement, so that the importer only logs a warning instead of raising an exception when the preset is absent. The change will be available in Vesper 0.4.8, coming soon.

**Comment by HaroldMills** [comment]
Fixed in 414746f7d1708a846cfc5b8d8af6007997631434.

**Comment by RichardLitt** [comment]
Thanks, @HaroldMills. Adding that file seems to get rid of that error.
